concourse/scripts/shake_concourse_volumes.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+cat <<"EOF"
+
+This operation will restart concourse and dropping all the volumes...
+
+                 ,-.
+       _,.      /  /          ,--.!,
+      ; \____,-==-._  )    __/   -*-
+      //_    `----' {+>  ,d08b.  '|`
+      `  `'--/  /-'`(    0088MM
+            /  /         `9MMP'       BLOW THAT CONCOURSE!!!
+      dew   `='       hjm
+                               via http://ascii.co.uk/art/plane
+                               and http://ascii.co.uk/art/bomb
+
+EOF
+
+set -e
+
+sudo -v -p "Type sudo password to continue: "
+
+echo "Stopping all the concourse services..."
+for i in atc beacon tsa baggageclaim garden ; do
+   sudo -i /var/vcap/bosh/bin/monit stop $i;
+done
+sleep 10
+
+echo "Killing all runc containers..."
+sudo pkill -f runc # Be sure that all containers are stopped
+sleep 10
+
+echo "Flushing all iptables rules from previus containers..."
+sudo iptables -F
+sudo iptables -X
+sudo iptables -t nat -F
+sudo iptables -t nat -X
+
+echo "Deleting all baggageclaim volumes and garden data"
+if mount | grep -q /var/vcap/data/baggageclaim/volumes; then
+   sudo umount -fl /var/vcap/data/baggageclaim/volumes
+fi
+if mount | grep -q /var/vcap/data/baggageclaim/volumes; then
+   sudo umount -fl /var/vcap/data/garden/graph
```

Comment:
I've also ran into what looks like a race condition here. I fixed the script to have the correct path, and got an error:
```
umount: /var/vcap/data/garden/graph: not mounted
```
I think this directory gets automatically unmounted by something when garden is stopped, and the last container has exited. We might need to do something more robust after the `pkill` above.